.drone.nightly.yml

```diff
@@ -0,0 +1,41 @@
+clone:
+  path: github.com/vmware/vic
+  tags: true
```

Comment:
I don't think you need this clone section since you are installing from bintray, have you tried testing without it?